app/src/main/java/com/framgia/wsm/data/source/RequestRepository.java

```diff
@@ -113,7 +113,7 @@ public RequestRepository(RequestRemoteDataSource remoteDataSource) {
         requestOverTime.setCreatedAt("23/06/2017");
         requestOverTime.setFromTime("21/06/2017 18:00");
         requestOverTime.setToTime("22/06/2017 19:00");
-        requestOverTime.setStatus(1);
+        requestOverTime.setStatus("pending");
```

Comment:
ERROR: [WrongConstant] Must be one of: StatusCode.ACCEPT_CODE, StatusCode.PENDING_CODE, StatusCode.REJECT_CODE
 Ensures that when parameter in a method only allows a specific set of constants, calls obey those rules.